meinberlin/assets/scss/components/_phases.scss

```diff
@@ -0,0 +1,32 @@
+.phaseList {
+    text-align: center;
+
+    ul {
+        margin: 0;
+        padding: 0;
+    }
+
+    li {
+        display: inline-block;
+    }
```

Comment:
These descendanct selectors are problematic because they also apply to all lists inside of the popup. We should refactor this later.